src/natcap/invest/habitat_quality.py

```diff
@@ -829,7 +896,8 @@ def total_degradation(*arrays):
 
 
 def _compute_rarity_operation(
-        base_lulc_path_band, lulc_path_band, new_cover_path, rarity_path):
+        base_lulc_path_band, lulc_path_band, new_cover_path,
+        rarity_raster_path, rarity_csv_path):
```

Comment:
Might be worth adding to the docstring that this function generates a raster and csv output.